src/QMCWaveFunctions/Fermion/SlaterDetBuilder.h

```diff
@@ -106,23 +106,47 @@ class SlaterDetBuilder : public WaveFunctionComponentBuilder
   template<typename VT,
            std::enable_if_t<(std::is_same<VT, ValueType>::value) &&
                             (std::is_floating_point<VT>::value), int> = 0>
-  void readCoeffs(hdf_archive& hin, std::vector<VT>& ci_coeff, size_t n_dets)
+  void readCoeffs(hdf_archive& hin, std::vector<VT>& ci_coeff, size_t n_dets,int ext_level)
   {
-    hin.read(ci_coeff, "Coeff");
+    std::string extVar;
+    extVar="Coeff_"+std::to_string(ext_level-1);
+    ///Determinant coeffs are stored in Coeff_N where N=1 for the ground state and N=2
+    ///Would be the first excited state.
+    ///Backwards compatibility: Old format assums "Coeff"
+    if (!hin.readEntry(ci_coeff,extVar))
```

Comment:
If we define Coeff for ground and add _x only for x =1, 2, 3. There is no fallback needed.

Reply:
this is fixed

Reply:
Is it necessary to update the h5 since Coeff_0 doesn't exist any more?
A side question. Does the h5 for testing included in this PR contains both ground and first excited states?